### PR TITLE
Edit category

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Update category
- Brower test
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/admin/pages/categories/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        @endif
+        @if (session('message'))
+          <div class="alert bg-green alert-dismissible" role="alert">
+            <button type="button" class="close" data-dismiss="alert"
+              aria-label="Close">
+              <span aria-hidden="true">×</span>
+            </button>
+            {{session('message')}}
+          </div>
+			  @endif
```
[the remark this answers is not in the document]
Comment (at-vyle) [reply resources/views/admin/pages/categories/edit.blade.php]: DONE :))

File: tests/Browser/Admin/Categories/UpdateCategoryTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function test_edit_child_category_success()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs($this->user)
+                    ->visit(new UpdateChildCategory)
+                    ->type('name', 'Trà Sửa Thái Đỏ')
+                    ->press('Edit')
+                    ->assertPathIs('/admin/categories/1/show-child')
+                    ->assertSee(__('category.admin.message.edit'));
```
Comment (at-quangtran) [inline tests/Browser/Admin/Categories/UpdateCategoryTest.php]: ko check db update giá trị đúng chưa hả?

File: tests/Browser/Admin/Categories/UpdateCategoryTest.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function test_edit_parent_category_success()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs($this->user)
+                    ->visit(new UpdateCategory)
+                    ->type('name', 'Trà Sửa Ngon')
+                    ->press('Edit')
+                    ->assertPathIs('/admin/categories')
+                    ->assertSee(__('category.admin.message.edit'));
```
Comment (at-quangtran) [inline tests/Browser/Admin/Categories/UpdateCategoryTest.php]: ko check db update giá trị đúng chưa hả?

File: app/Models/Category.php [hunk 4]
```diff
@@ -42,7 +42,7 @@ public function products()
      *
      * @return int
      */
-    public function countChild($id)
+    public static function countChild($id)
```
Comment (at-quangtran) [inline app/Models/Category.php]: vì sao dùng static?

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**